platform/linux-generic/include/odp_queue_internal.h

```diff
@@ -77,19 +63,18 @@ struct queue_entry_s {
 	char              name[ODP_QUEUE_NAME_LEN];
 };
 
-union queue_entry_u {
+typedef union queue_entry_u {
 	struct queue_entry_s s;
 	uint8_t pad[ROUNDUP_CACHE_LINE(sizeof(struct queue_entry_s))];
-};
-
+} queue_entry_t;
 
 queue_entry_t *get_qentry(uint32_t queue_id);
 
-int queue_enq(queue_entry_t *queue, odp_buffer_hdr_t *buf_hdr);
-odp_buffer_hdr_t *queue_deq(queue_entry_t *queue);
+int _queue_enq(queue_t handle, odp_buffer_hdr_t *buf_hdr);
+odp_buffer_hdr_t *_queue_deq(queue_t handle);
```

Comment:
There's really no need for the _ prefix then. The base names can be used as is, no? You're just changing their type signatures, which seems sufficient.